use_case_guide/multi_tenant.rst

```diff
@@ -0,0 +1,538 @@
+    site_url text NOT NULL,
+    cost_per_impression_usd numeric(20,10),
+    user_ip inet NOT NULL,
+    user_data jsonb NOT NULL,
+    PRIMARY KEY (company_id, id),       -- added
+    FOREIGN KEY (company_id, ad_id)     -- added
+      REFERENCES ads (company_id, id)
+  );
+
+You can learn more about migrating your own data model in :ref:`multi-tenant schema migration <mt_schema_migration>`.
+
+Try it Yourself
+~~~~~~~~~~~~~~~
+
+.. note::
+
+  This guide is designed so you can follow along in your own Citus database. Use one of these alternatives to spin up a database:
+
+  * Run Citus locally using :ref:`single_machine_docker`, or
+  * Provision a cluster using `Citus Cloud <https://console.citusdata.com/users/sign_up>`_
+
+  You'll run the SQL commands using psql:
+
+  * **Docker**: :code:`docker exec -it citus_master psql -U postgres`
+  * **Cloud**: :code:`psql "connection-string"` where the connection string for your formation is available in the Cloud Console.
+
+  In either case psql will be connected to the coordinator node for the cluster.
+
+At this point feel free to follow along in your own Citus cluster by `downloading <https://examples.citusdata.com/mt_ref_arch/schema.sql>`_ and executing the SQL to create the schema. Once the schema is ready, we can tell Citus to create shards on the workers. From the coordinator node, run:
+
+::
+
+  SELECT create_distributed_table('companies',   'id');
+  SELECT create_distributed_table('campaigns',   'company_id');
+  SELECT create_distributed_table('ads',         'company_id');
+  SELECT create_distributed_table('clicks',      'company_id');
+  SELECT create_distributed_table('impressions', 'company_id');
+
+The :ref:`create_distributed_table` function informs Citus that a table should be distributed among nodes and that future incoming queries to those tables should be planned for distributed execution. The function also creates shards for the table on worker nodes, which are low-level units of data storage Citus uses to assign data to nodes.
+
+The next step is loading sample data into the cluster from the command line.
+
+.. code-block:: bash
+
+  # download and ingest datasets from the shell
+
+  for dataset in companies campaigns ads clicks impressions geo_ips; do
+    curl -O https://examples.citusdata.com/mt_ref_arch/${dataset}.csv
+  done
+
+.. note::
+
+  **If you are using Docker,** you should use the :code:`docker cp` command to copy the files into the Docker container.
+
+  .. code-block:: bash
+
+    for dataset in companies campaigns ads clicks impressions geo_ips; do
+      docker cp ${dataset}.csv citus_master:.
+    done
+
+Being an extension of PostgreSQL, Citus supports bulk loading with the COPY command. Use it to ingest the data you downloaded, and make sure that you specify the correct file path if you downloaded the file to some other location. Back inside psql run this:
+
+.. code-block:: psql
+
+  \copy companies from 'companies.csv' with csv
```

Comment:
Minor note. This seems to highlight only the first copy for me. The others appear grayed out.
<img width="447" alt="screen shot 2017-06-30 at 11 50 27 am" src="https://user-images.githubusercontent.com/2596363/27749858-3698b2d0-5d8a-11e7-933d-c3ed6a0f40be.png">